src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/update/UpdateOperation.java

```diff
@@ -59,6 +59,15 @@ protected static String validateUpdatePath(UpdateOperator oper, String path) {
           ErrorCode.UNSUPPORTED_UPDATE_FOR_VECTOR.getMessage() + ": " + oper.operator());
     }
 
+    if (DocumentConstants.Fields.VECTOR_EMBEDDING_TEXT_FIELD.equals(path)
+        && !(oper.operator().equals("$set")
+            || oper.operator().equals("$unset")
```

Comment:
Does `$unset` make sense with `$vectorize`? Looks like it should be equivalent to `$unset` on `$vector`. What happens if you `$unset` `$vector` but `$set` on `$vectorize`?

Reply:
Good point. I will add validation that vectorize and vector can't be used together.


> Does `$unset` make sense with `$vectorize`?

Not Sure if the user uses vectorize to load, thought will use the same to unset. I can remove it if you think it's not required.

Reply:
Should `$unset` on `$vector` automatically clear `$vectorize` field if one exists (now that we store it)?

Reply:
In case set or unset come with $vector field, the logic should remove existing $vectorize field in the main document?